Source/Modules/javascript.cxx

```diff
@@ -768,6 +786,22 @@ int JSEmitter::emitWrapperFunction(Node *n) {
   return ret;
 }
 
+int JSEmitter::emitNativeFunction(Node *n) {
+  String *wrapname = Getattr(n, "wrap:name");
+  // ismember never seems to be the case;
+  // it is technically possible to add native member functions,
+  // just not at the moment? leaving this as an option for later;
+  // the code will automatically defaulting to static space
+  if (GetFlag(n, "ismember") != 0)
+    Setattr(n, "feature:extend", "1"); // member space
```

Comment:
Setting this flag is reserved for the parser when parsing %extend. So this code needs to be reimplemented. %extend has nothing to do with %native.

Are you trying to filter out non-static member functions used with %native? Does one of the testcases fail without this flag setting?

Reply:
That is what I was asking about several months ago on the mailing list. (June 26)
When I wrote the function implementation I referenced the python implementation, which at the time also used it.

The python implementation had set the value of `feature:extend` to 1, which I observed made the function a member function. However, when writing the test case for the JS impl., I saw that Java's function were global / static. Hence why I was confused.

Looking over the python implementation now, it looks like the line was removed as a correction. My JS implementation should work without the modification of `feature:extend`. I will remove it.

Reply:
Thanks, I'll go with this then. For the future, this is a public code base, please use commit message that say something more useful than "Dev Checkpoint".

Reply:
Sorry; I was under the impression that I would eventually need to flatten all my commits into one before the PR was merged. I didn't want to waste time naming every little fix if the commits were just going to be absorbed, otherwise I would have given them each unique names.